build/yaml/crd/new/crd.nsx.vmware.com_securitypolicies.yaml

```diff
@@ -0,0 +1,660 @@
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
I feel this new one should be put in crd folder.

Reply:
ok